<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_MidiMapper</class>
 <widget class="QDialog" name="Dialog_MidiMapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI Mapper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0795909, y1:0.119, x2:0.983, y2:0.998455, stop:0 rgba(117, 117, 117, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_MidiMapperCh_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_14">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_15">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_12">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_MidiMapperOK">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_11">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>107</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>267</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 13</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>227</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 11</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_13">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_16">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>33</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_15">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 15</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>307</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>187</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>67</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_10">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>67</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_12">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_MidiMapperCancel">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>147</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>27</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_16">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 16</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_MidiMapperCh_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>107</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-color: rgb(0, 0, 0);
background-color: rgb(209, 209, 209);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_MidiMapperCh_14">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CH 14</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
